tests/ListPool.UnitTests/ListPoolExtensionsTests.cs

```diff
@@ -33,7 +33,7 @@ public void ToListPool_when_source_is_null_throw_ArgumentNullException()
             IEnumerable<int> source = null;
             string expectedName = nameof(source);
 
-            var exception = Assert.Throws<ArgumentNullException>(() => source.ToListPool());
+            ArgumentNullException exception = Assert.Throws<ArgumentNullException>(() => source.ToListPool());
```

Comment:
Why not use **var** here?

Reply:
recommendation after merge this: https://github.com/faustodavid/ListPool/blob/master/.editorconfig